requirements.txt

```diff
@@ -1,17 +1,22 @@
 # As we approach deployment, we should probably move to different requirements files for local vs. production
 # See Chapt 5.5 in Two Scoops of Django
-boto3==1.13.16
+# Unipath==1.1s
+# python-votesmart=0.3.3  # Using local version instead since 0.3.3 not Py 3 compatible
+# sendgrid-django==4.2.0
+Pillow==9.3.0
```

Comment:
Hi @SailingSteve, I'm curious why you moved these capitalized packages to the top of the list, instead of leaving them in alphabetical order?